src/tests/utils/insert-data.ts

```diff
@@ -23,10 +26,12 @@ async function insertUsers(db: SequelizeOrm): Promise<void> {
 async function insertUserSessions(db: SequelizeOrm): Promise<void> {
   for (const userSession of userSessions) {
     await db.connection.query(`INSERT INTO public.user_sessions (id, "user_id", "refresh_token", "refresh_token_expires_at") VALUES (${userSession.id}, ${userSession.user_id}, '${userSession.refresh_token}', '${userSession.refresh_token_expires_at}')`);
+    await db.connection.query(`UPDATE public.user_sessions SET "refresh_token_expires_at" = '${date}' WHERE id = 5;
+    `);
   }
 }
```

Comment:
i think you should update `user-sessions.json` before inserting into database, not update db data

Reply:
Tanya told me to improve _insertUserSessions()_ function, to don't write new expiration date in _user-sessions.json_  every time. 
P.S. Your variant I have already done in last version.